libs/dh/market-participant/actors/feature-actors/src/lib/status-badge/dh-actor-status-badge.component.ts

```diff
@@ -43,5 +43,5 @@ import { DhEmDashFallbackPipe } from '@energinet-datahub/dh/shared/ui-util';
   imports: [TranslocoDirective, WattBadgeComponent, DhEmDashFallbackPipe],
 })
 export class DhActorStatusBadgeComponent {
-  @Input({ required: true }) status: ActorStatus | null | undefined = null;
+  status = input.required<ActorStatus | null | undefined>();
```

Comment:
Question: If it is required, should it then be allowed to be null or undefined as well?

Reply:
Is where a good reason for it to also be null or undefined?

Reply:
According to the types, it can be `undefined`. So maybe we can get rid of the `null`